nemo/core/classes/modelPT.py

```diff
@@ -1234,6 +1233,18 @@ def set_world_size(self, trainer: Trainer):
         app_state = AppState()
         app_state.world_size = self.world_size
 
+    def summarize(self, max_depth: int = 1) -> model_summary.ModelSummary:
```

Comment:
Why summerize is wrapped here?

Reply:
It is being deprecated in PTL 1.7.0 for exactly this method lol, thought might as well patch it now than have it break later.